samples/other/copy-between-azure-and-s3/src/main/java/org/eclipse/dataspaceconnector/transfer/BlobToS3DataFlowController.java

```diff
@@ -34,15 +34,15 @@ public BlobToS3DataFlowController(Vault vault, Monitor monitor, TypeManager type
     @Override
     public boolean canHandle(DataRequest dataRequest) {
```

Comment:
this entire class exists in an almost identical form a second time in the `...extensions/transfer` package. I assume this is an oversight?

Reply:
I will try to use sample 5 and removed all the `DataFlowController` code.